pkg/cluster/cluster.go

```diff
@@ -87,6 +88,8 @@ type Cluster struct {
 	backupDir string
 
 	gc *garbagecollection.GC
+
+	etcdHttpClient *http.Client
```

Comment:
this client is used to talk to the backup sidecar, right? can we rename it to backupHTTPClient?